test/plots/areaPlotTests.ts

```diff
@@ -2,162 +2,181 @@
+        yScale.padProportion(0.1);
+        let constantY0 = 30;
+        yScale.addIncludedValuesProvider((scale: Plottable.Scales.Linear) => [constantY0, constantY0 + 10]);
+        plot.y0(constantY0);
+        plot.addDataset(new Plottable.Dataset([{ x: 0, y: constantY0 + 5 }]));
+        plot.renderTo(svg);
+        assert.strictEqual(yScale.domain()[0], constantY0, "y Scale doesn't pad beyond 0 when used in a Plots.Area");
+        svg.remove();
+      });
     });
 
-    beforeEach(() => {
-      svg = TestMethods.generateSVG(500, 500);
-      simpleDataset = new Plottable.Dataset(twoPointData);
-      areaPlot = new Plottable.Plots.Area<number>();
-      areaPlot.addDataset(simpleDataset);
-      areaPlot.x(xAccessor, xScale)
-              .y(yAccessor, yScale);
-      areaPlot.y0(y0Accessor)
-              .attr("fill", fillAccessor)
-              .attr("stroke", colorAccessor)
-              .renderTo(svg);
-      renderArea = (<any> areaPlot)._renderArea;
-    });
+    describe("Rendering", () => {
+      let svg: d3.Selection<void>;
+      let xScale: Plottable.Scales.Linear;
+      let yScale: Plottable.Scales.Linear;
+      let areaPlot: Plottable.Plots.Area<number>;
+      let fill = "steelblue";
+
+      beforeEach(() => {
+        xScale = new Plottable.Scales.Linear();
+        yScale = new Plottable.Scales.Linear();
+        svg = TestMethods.generateSVG();
+        let twoPointData = [{x: 0, y: 0}, {x: 1, y: 1}];
+        let simpleDataset = new Plottable.Dataset(twoPointData);
+        areaPlot = new Plottable.Plots.Area<number>();
+        areaPlot.addDataset(simpleDataset);
+        areaPlot.x((d: any) => d.x, xScale)
+                .y((d: any) => d.y, yScale);
+        areaPlot.y0(() => 0)
+                .attr("fill", () => fill)
+                .attr("stroke", (d, i) => d3.rgb(d.x, d.y, i).toString())
+                .renderTo(svg);
+      });
 
-    it("draws area and line correctly", () => {
-      let areaPath = renderArea.select(".area");
-      assert.strictEqual(TestMethods.normalizePath(areaPath.attr("d")), "M0,500L500,0L500,500L0,500Z", "area d was set correctly");
-      assert.strictEqual(areaPath.attr("fill"), "steelblue", "area fill was set correctly");
-      let areaComputedStyle = window.getComputedStyle(<Element> areaPath.node());
-      assert.strictEqual(areaComputedStyle.stroke, "none", "area stroke renders as \"none\"");
-
-      let linePath = renderArea.select(".line");
-      assert.strictEqual(TestMethods.normalizePath(linePath.attr("d")), "M0,500L500,0", "line d was set correctly");
-      assert.strictEqual(linePath.attr("stroke"), "#000000", "line stroke was set correctly");
-      let lineComputedStyle = window.getComputedStyle(<Element> linePath.node());
-      assert.strictEqual(lineComputedStyle.fill, "none", "line fill renders as \"none\"");
-      svg.remove();
-    });
+      it("draws area and line correctly", () => {
```

Comment:
can we be more exact with what draws correctly means?

it seems it involves that the line path is the data points
it also seems that the area has the specified fill
it also seems that the line is not filled
it also seems that the area has no stroke
it also seems that the line has the specified stroke

I personally think this may be different tests since the main problem is that "draws correctly" is incredibly vague